docs/DeveloperGuide.md

```diff
@@ -772,13 +814,31 @@ testers are expected to do more *exploratory* testing.
         1. Test case 1: `dark`
             - Testing for: Successful command execution
             - Expected: UI updates to dark mode
-1. Set UI to light mode from light mode
+1. Set UI to dark mode from dark mode
     1. Prerequisites: UI currently in dark mode (can be switched using the `dark` command)
     2. Test Cases
         1. Test case 1: `dark`
             - Testing for: Successful command execution
             - Expected: UI updates to dark mode
 
+### Moving to another tab in NeoBook
+1. Changing to a different tab
+   1. Prerequisites: NeoBook currently on address book tab.
+   2. Test Cases
+      1. Test case 1: `tab 2`
+         - Testing for: Successful command execution
+         - Expected: Switch to Events tab
+      2. Test case 2: `tab 3`
+       - Testing for: Successful command execution
+       - Expected: Switch to Me tab
+2. Changing to same tab
+   1. Prerequisites: NeoBook currently on address book tab.
+   2. Test Cases
+      1. Test case 1: `tab 1`
+         - Testing for: Successful command execution
+         - Expected: No changes made because Neobook is already on desired tab. Warning message appears in
+         status to remind user that he/she is already on the tab.
+
```

Comment:
```suggestion
### Moving to another tab in NeoBook
1. Changing to a different tab

   1. Prerequisites: NeoBook currently on address book tab.
   
   2. Test Cases
      1. Test case 1: `tab 2`
         - Testing for: Successful command execution
         - Expected: Switch to Events tab
      2. Test case 2: `tab 3`
       - Testing for: Successful command execution
       - Expected: Switch to Me tab
       
2. Changing to same tab

   1. Prerequisites: NeoBook currently on address book tab.
   
   2. Test Cases
      1. Test case 1: `tab 1`
         - Testing for: Successful command execution
         - Expected: No changes made because Neobook is already on desired tab. Warning message appears in
         status to remind user that he/she is already on the tab.

```